Makefile

```diff
@@ -54,6 +54,12 @@ clippy: nsm-api-${STABLE}
 eif_dir:
 	mkdir -p eifs/${HOST_MACHINE}/
 
+command-executer: command-executer
```

Comment:
I'm not sure I understand why the command-executor is needed here? The two tests included here can be used as direct targets rather than requiring an intermediary.

Reply:
The command-executer is used to launch the tests inside the enclave and also for retrieving the output/ return code from the tests.